frontend/src/types/CamperTypes.ts

```diff
@@ -1,6 +1,7 @@
 export type Camper = {
   id: string;
   // campSession: CampSession;
+  campSession: string;
```

Comment:
`campSession` was originally commented out (so we've never used this before), but logging a camper shows that `campSession` is of type string, not type CampSession, so I decided to keep it as string.

Reply:
That's true, I do see expressions like `campSession: camper.campSession ? camper.campSession.toString() : ""` in a lot of places where we're GETting campers from endpoints so it looks valid to me.